core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -555,7 +589,7 @@ class EMConfig {
     var customNaming = false
 
     /*
-        You need to decode it if you want to know what it says...
+        You need to decode it if you want to know what is says...
```

Comment:
i think i did recently fix this typo, but that was likely after last time you rebased/merged from master?

Reply:
it is a mistake I made. when I solved the conflicts, I accepted my version. the problem is fixed now.